### PR TITLE
Add stylelint 13 to peer dependency range.

### PR DESCRIPTION
**Pull request by mario-fehr**

### CODE REVIEW

**Review by alexander-schranz — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -22,13 +22,13 @@
...
+    "stylelint-config-standard": "^20.0.0",
     "stylelint-scss": "^3.0.0"
   },
   "devDependencies": {
-    "stylelint": "^10.0.0"
+    "stylelint": "^13.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.3.0 || ^9.0.0 || ^10.0.0"
+    "stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
```
Comment (alexander-schranz) [inline package.json]: why not allow the lower versions we did normally just add the new stylelint versions?

Comment (mario-fehr) [reply package.json]: @alexander-schranz do we realy need to support stylelint version <10 with a new version? `stylelint-config-standard` dropt support for stylelint version <10 with release `19.0.0` and this PR was made to get rid of the warnings that  `stylelint-config-standard` version `18.2.0` does no support stylelint version >10.0.0

Comment (alexander-schranz) [reply package.json]: > do we realy need to support stylelint version

else we would need todo a new major release which will actually every project then update to it. So the question is more does it hurt to support older stylelint versions?

Comment (mario-fehr) [reply package.json]: @alexander-schranz I will check.

Comment (mario-fehr) [reply package.json]: @alexander-schranz I added the older versions back. Not 100% sure how `peerDependencies` but I think it should work.

**Comment by alexander-schranz** [comment]
Released as 1.1.5

**Pull request merged**